sdks/python/apache_beam/utils/pipeline_options_validator_test.py

```diff
@@ -87,7 +87,7 @@ def get_validator(temp_location):
       return validator
 
     test_cases = [
-        {'temp_location': None, 'errors': ['temp_location']},
```

Comment:
This test matrix needs to be extended with a few staging_location, temp_location mixed inputs. Because there is a dependency between them now.